### ISSUE TITLE
Reopen: Training vanilla SimCLR

### ISSUE THREAD

**Issue by hyun78** [alinlab/CSI#18]
#16 
Hi, I run the following code.
for training:
CUDA_VISIBLE_DEVICES=0,1,2,3 python3 -m torch.distributed.launch --nproc_per_node=4 train.py --dataset cifar10 --model resnet18 --mode simclr --shift_trans_type none --batch_size 128 --one_class_idx 0 --optimizer adam --suffix new
for evaluation:
python3 eval.py --mode ood_pre --dataset cifar10 --model resnet18 --ood_score simclr --shift_trans_type none --print_score --ood_samples 10 --resize_factor 0.54 --resize_fix --one_class_idx 0 --load_path <load_path>

And I got a different result.
![image](https://user-images.githubusercontent.com/19701838/103885273-18fef780-5123-11eb-813f-4f45ce6fca6d.png)

I think it should be 87.9 (in the paper, Table 7a). It may be lower due to the optimizer, but it is too low.

Is there any problem with my command?
Could you provide sample codes to reproduce the Table 7a result?
Thanks

**Comment by Jihoon-Tack** [comment]
Hi!

The command seems okay.. the only thing that is different is the optimizer.
Maybe you should use another learning rate value (e.g., 1e-3, 1e-4), since we commonly use a smaller learning rate or `adam` optimizer. And also removing the learning rate scheduler might help for `adam`(also the learning rate warmup).

Maybe using `lars` will be a better option, since we have followed the same hyperparameters from SimCLR.

Best,
Jihoon

**Comment by Jihoon-Tack** [comment]
If you have any questions or troubles, please feel free to reopen the issue
